### PR TITLE
make array

### PR DESCRIPTION
**Pull request by Chityanj**

### CODE REVIEW

File: maintainers.json [hunk 1]
```diff
@@ -1,289 +1,243 @@
...
     {
+        "codename": "taimen",
         "brand": "Google",
         "name": "Pixel 2 XL",
         "maintainer": "Yomatsun",
         "telegram": "Yomatsun",
         "xda_thread": ""
-    },
-
-    "bretzel":
```
Comment (SebaUbuntu) [inline maintainers.json:272]: Why removing these devices?

**Comment by ethan-halsall** [comment]
I just do not see why you need to do this

**Comment by ethan-halsall** [comment]
Also, the bot must be rewritten

**Review by ethan-halsall — CHANGES_REQUESTED** [review]
Make sure to update all apis - bot which reads this json

**Comment by SebaUbuntu** [comment]
An hypothetical website can work without array

**Pull request closed without merging**